QuasistaticBrownianThermalNoise/QuasistaticBrownianThermalNoise.cpp

```diff
@@ -608,10 +696,36 @@ CylTransFunc::CylTransFunc(double coatThick, double halfCylThick):
 	      Y_Iso_Ta2O5[i][j][k][l] += lame_mu_Ta2O5;
 	    }
 	    
-	  }
-	}
+              }
+          }
       }
-    }  
+    }
+    
+    //Set Y_Ta2O5 in terms of the bulk and shear moduli
+    for(int i=0;i<dim;++i) {
+        for(int j=0;j<dim;++j) {
+            for(int k=0;k<dim;++k) {
+                for(int l=0;l<dim;++l) {
+                    
+            //Y_{ijkl} = lambda dij dkl + mu dik djl + mu dil djk
+            if(i==j && k==l) {
+	      Y_Iso_Ta2O5_Phi[i][j][k][l] += (((lame_lambda_Ta2O5 + (lame_mu_Ta2O5 * (2.0/3.0)))  * lossPhi_Ta2O5_bulk) - (((2.0/3.0) * lame_mu_Ta2O5) * lossPhi_Ta2O5_shear) );
+                }
+            
+            if(i==k && j==l) {
+	      Y_Iso_Ta2O5_Phi[i][j][k][l] += (lame_mu_Ta2O5 * lossPhi_Ta2O5_shear);
+                }
+                      
+            if(i==l && j==k) {
+	      Y_Iso_Ta2O5_Phi[i][j][k][l] += (lame_mu_Ta2O5 * lossPhi_Ta2O5_shear);
+                }
+                      
+                  }
```

Comment:
Make a separate function that takes lame_mu and lame_lambda, bulk loss, shear loss, and return yijkl. Then call that function for each specific material.